Modules/Filtering/LabelMap/include/itkStatisticsLabelMapFilter.h

```diff
@@ -137,9 +137,11 @@ class ITK_TEMPLATE_EXPORT StatisticsLabelMapFilter
   static constexpr unsigned int
   GetDefaultNumberOfBins()
   {
-    return NumericTraits<FeatureImagePixelType>::IsInteger && sizeof(FeatureImagePixelType) <= 2
-             ? 1 << (8 * sizeof(FeatureImagePixelType))
-             : 128;
+    constexpr unsigned short bitsShift =
+      std::min(8 * sizeof(typename Self::FeatureImagePixelType), sizeof(unsigned int) - 1);
```

Comment:
I believe that an `8` is missing; it should be:
```c++
std::min(8 * sizeof(typename Self::FeatureImagePixelType), 8 * sizeof(unsigned int) - 1);
```

Reply:
ugh, that was sloppy.

Reply:
Why the implicit downsize to `short`? ie why not `size_t` (what `sizeof` returns)?

Reply:
done